test/e2e/storage/csi_mock/csi_kubelet_restart.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package csi_mock
```

Comment:
@carlory It appears that the CI is failing because of this package name, is it possible to get this fixed? I see that other cherry-picks are using `csimock` instead

Reply:
@xmudrii updated.